### ISSUE TITLE
[ZigBee] Add support for Groups and Scenes cluster (configuration in the meshdriver)

### ISSUE THREAD

**Issue by TedTolboom** [athombv/homey#2025]
**Metadata**
* **Homey Version**:1.5.x

Current ZigBee (meshdriver) implementation does not support the (configuration of the) Groups and Scenes cluster. 

From ZigBee specification:
> **Groups:** Attributes and commands for allocating a device to one or more of a number
> of groups of devices, where each group is addressable by a group address.
> **Scenes:** Attributes and commands for setting up and recalling a number of scenes for
> a device. Each scene corresponds to a set of stored values of specified device
> attributes. 

This is one of the main advantages of the Zigbee protocol; enabling to switch on the lights directly (Homey to light) and simultaneously; like done with e.g. the Hue hub. In addition, this would also enable the use of a remote (Tradfri dimmer) in combination with both Homey and the light itself.

I checked homey/issues as well as the meshdriver repository; no similar (feature) request found.

**Comment by bvdbos** [comment]
dupe of #1940 ?

**Comment by TedTolboom** [comment]
No, two different issues that need to be solved to complete the Zigbee
implementation
